### PR TITLE
Improve design for community site

### PR DESCRIPTION
**Pull request by YoonSuhyeok**
firefox-devtools/ux#120

I edit text (Master CSS->Master Grid)

After a few days of doing it, I learned about the contribution.

violasong! Thanks for this opportunity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,24 @@ Help us maintain an inclusive environment by following our [code of conduct](COD
 
 # About DevTools
 
+## Highlights
+
+* [Inactive CSS](https://hacks.mozilla.org/2019/10/firefox-70-a-bountiful-release-for-all/#developertools):
+Firefox DevTools now grays out CSS declarations that don’t have an effect on the page. When you hover over the info icon, you’ll see a useful message about why the CSS is not being applied, including a hint about how to fix the problem.
+![Inactive_CSS](https://www.mozilla.org/media/img/firefox/developer/hero-inactive-css.40a2e0b8ddb1.png) 
+
+* [Master Gird](https://mozilladevelopers.github.io/playground/css-grid/):
```
Comment (violasong) [inline README.md:42]: Ah, noticed a typo: this should say Grid not Gird

File: README.md [hunk 2]
```diff
@@ -33,6 +33,24 @@ Help us maintain an inclusive environment by following our [code of conduct](COD
...
+Grid Firefox is the only browser with tools built specifically for building and designing with CSS Grid. These tools allow you to visualize the grid, display associated area names, preview transformations on the grid and much more.
```
Comment (violasong) [inline README.md:43]: Should remove extra "Grid" text at the beginning of this sentence

File: README.md [hunk 3]
```diff
@@ -33,6 +33,24 @@ Help us maintain an inclusive environment by following our [code of conduct](COD
 
 # About DevTools
 
+## Highlights
+
+* [Inactive CSS](https://hacks.mozilla.org/2019/10/firefox-70-a-bountiful-release-for-all/#developertools):
+Firefox DevTools now grays out CSS declarations that don’t have an effect on the page. When you hover over the info icon, you’ll see a useful message about why the CSS is not being applied, including a hint about how to fix the problem.
+![Inactive_CSS](https://www.mozilla.org/media/img/firefox/developer/hero-inactive-css.40a2e0b8ddb1.png)
```
Comment (violasong) [inline README.md:40]: Let's remove all the images for now, until we know what the design will be with the other work that's happening.

**Comment by janodvarko** [comment]
The site is now completely redesigned, closing.

**Pull request closed without merging**